SharedPackages/VPN/Sources/VPN/NetworkProtectionDeviceManager.swift

```diff
@@ -283,7 +283,6 @@ public actor NetworkProtectionDeviceManager: NetworkProtectionDeviceManagement {
             default:
                 throw unmanagedSubscriptionError(underlyingError: error)
             }
-            throw error
```

Comment:
This was giving a warning since it will never throw (due to the `default` entry above)